src/Config/Validator/Deploy/ServiceEol/Notices.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
This class is a duplicate of `Warnings` class except for log level.
You can leave only one general Class with configured log level and pass log level through symfony di.
Create two services of one class in `services.xml` and pass them different log levels.